backend/src/nodes/nodes/rest/img2img.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+
+import numpy as np
+
+from . import category as RESTCategory
+from ...impl.rest import (
+    decode_base64_image,
+    SamplerName,
+    STABLE_DIFFUSION_IMG2IMG_URL,
+    post_async,
+    encode_base64_image,
+)
+from ...node_base import AsyncNodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import (
+    TextInput,
+    NumberInput,
+    EnumInput,
+    ImageInput,
+)
+from ...properties.outputs import LargeImageOutput
+
+
+@NodeFactory.register("chainner:rest:sd_img2img")
+class Img2Img(AsyncNodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Modify an image using an external Stable Diffusion service"
+        self.inputs = [
+            ImageInput(),
+            TextInput("Prompt", default="an astronaut riding a horse"),
+            TextInput("Negative Prompt", default=""),
+            NumberInput(
+                "Denoising Strength",
+                minimum=0,
+                default=0.75,
+                maximum=1,
+                controls_step=0.1,
+                precision=4,
+            ),
+            NumberInput("Seed", minimum=0, default=42, maximum=4294967296),
```

Comment:
We have a group (=wrappers around inputs that add functionality) for seeds. It adds a button that when pressed generates a random 6-digit number.

![image](https://user-images.githubusercontent.com/20878432/214319644-d1058dc7-aa21-42f4-8eae-e5fe4f455024.png)

(The `group` function is imported from `node_base`.)

```suggestion
            group("seed")(
                NumberInput("Seed", minimum=0, default=42, maximum=4294967296)
            ),
```

Reply:
Cool!  That is a lot nicer